### PR TITLE
Fancy notifications

### PR DESCRIPTION
**Pull request by Earu**
Basically overrides default notification functions to have fancy new notification style.

### CODE REVIEW

File: CoolNotifications/autorun/client/CoolNotifs.lua [hunk 1]
```diff
@@ -0,0 +1,97 @@
+surface.CreateFont( "NoticeFont", {
+	font = "BITSTREAM VERA SANS", --Use the font-name which is shown to you by your operating system Font Viewer, not the file name
```
Comment (Techbot121) [inline CoolNotifications/autorun/client/CoolNotifs.lua]: make sure you use a font which is on gmod by default, other wise you have to add it manually
you can check them all out in `GarrysMod\garrysmod\resource\ClientScheme.res` under fonts

**Review by Techbot121 — CHANGES_REQUESTED** [review]

**Review by Techbot121 — COMMENTED** [review]
no changes?

File: CoolNotifications/autorun/client/CoolNotifs.lua [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	
+	notice:SetText(message)
+	notice:SetPos(ScrW(), ScrH() - (i - 1) * (notice:GetTall() + 4 	) + 4)
+	notice:SizeToContentsX()
+	notice:SetWide(notice:GetWide() + 64)
+	notice.start = CurTime() + 0.25
+	notice.endTime = CurTime() + delay
+	notice.id = i	
+	notice.OnRemove = function() 
+		notices[notice.id] = nil
```
Comment (Techbot121) [inline CoolNotifications/autorun/client/CoolNotifs.lua:56]: I think you should replace this with `table.remove(notices,notice.id)` but try it out first

**Review by Techbot121 — CHANGES_REQUESTED** [review]

File: CoolNotifications/autorun/client/CoolNotifs.lua (hunk 2 above)
Comment (Earu) [reply CoolNotifications/autorun/client/CoolNotifs.lua:56]: thats what I was doing earlier, but it turned out to be uh messy

Comment (Earu) [reply CoolNotifications/autorun/client/CoolNotifs.lua:56]: plus didnt remove value from the table sometimes

**Review by Techbot121 — APPROVED** [review]

**Pull request merged**